Sources/Queues/Application+Queues.swift

```diff
@@ -4,33 +4,31 @@ import Vapor
 import NIO
 
 extension Application {
-    /// The `Queues` object
+    /// The application-global ``Queues`` accessor.
     public var queues: Queues {
         .init(application: self)
     }
     
-    /// Represents a `Queues` configuration object
+    /// Contains global configuration for queues and provides methods for registering jobs and retrieving queues.
     public struct Queues {
-        
-        /// The provider of the `Queues` configuration
+        /// A provider for a ``Queues`` driver.
         public struct Provider {
-            let run: (Application) -> ()
+            let run: @Sendable (Application) -> ()
 
-            public init(_ run: @escaping (Application) -> ()) {
+            public init(_ run: @escaping @Sendable (Application) -> ()) {
                 self.run = run
             }
         }
 
-        final class Storage {
+        final class Storage: @unchecked Sendable {
```

Comment:
Is this actually safe? the `add(command:)` function does not synchronise access

Reply:
Oops, it is not, good catch!